api/api/model_endpoint_alerts_api.go

```diff
@@ -63,9 +61,9 @@ func (c *AlertsController) GetModelEndpointAlert(r *http.Request, vars map[strin
 	modelEndpointAlert, err := c.ModelEndpointAlertService.GetModelEndpointAlert(modelID, modelEndpointID)
 	if err != nil {
 		if gorm.IsRecordNotFoundError(err) {
-			return NotFound(fmt.Sprintf("GetModelEndpointAlert: Alert for model endpoint with id %s not found", modelEndpointID))
+			return NotFound(fmt.Sprintf("Model alert not found: %v", err))
```

Comment:
Should this be `fmt.Sprintf("Model endpoint alert not found: %v", err)`, just like in your changes to `UpdateModelEndpointAlert`?